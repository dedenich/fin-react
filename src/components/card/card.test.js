import React from 'react';
import renderer from 'react-test-renderer';

import Card from './card.jsx';


it(`Card renders correctly`, () => {
  const tree = renderer
  .create(
      <Card
        capture={`Поступления`}
      />
  )
  .toJSON();
  expect(tree).toMatchSnapshot();
});
