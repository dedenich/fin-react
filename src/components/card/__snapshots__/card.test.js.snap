// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
<div
  className="card-wrapper"
>
  <p
    className="capture"
  >
    Поступления
    :
  </p>
  <div
    className="card"
  />
</div>
`;
